### PR TITLE
adding taxons

### PR DESCRIPTION
**Pull request by sabrinatoro**
addresses #3855

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
@sabrinatoro I think it assigned me as a reviewer by default. Is this ready for review?

**Comment by sabrinatoro** [comment]
@nicolevasilevsky It is still a draft. I am having issues with importing and/or updating NCBITaxon, and I haven't had a chance to work on it today. I'll tag you when it is ready for review. Thank you for checking

**Comment by nicolevasilevsky** [comment]
sounds good! I could try to run the ncbi taxon import if it continues to give you trouble, let me know 😸

File: src/ontology/imports/ncbitaxon_terms.txt [hunk 1]
```diff
@@ -1427,3 +1438,34 @@ http://purl.obolibrary.org/obo/NCBITaxon_9922
 http://purl.obolibrary.org/obo/NCBITaxon_9925
 http://purl.obolibrary.org/obo/NCBITaxon_9963
 http://purl.obolibrary.org/obo/NCBITaxon_9989
+http_//purl.obolibrary.org/obo/NCBITaxon_119825
```
Comment (matentzn) [inline src/ontology/imports/ncbitaxon_terms.txt:1441]: Typo in URL! `_`!

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/ontology/imports/ncbitaxon_terms.txt (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/imports/ncbitaxon_terms.txt]: !!! Thank you for catching this! 
I have fixed the typos

**Comment by sabrinatoro** [comment]
@nicolevasilevsky and @matentzn 
This PR is finally ready for review (whenever you get a chance): it is a long time and a lot of jumping through hoops, but I think it is done.

**Comment by sabrinatoro** [comment]
@nicolevasilevsky 
This is failing because of the ISBN id 
I found examples in Mondo using these ids in this format, but it now looks like this is failing the checks. I don't know what the path forward is (ie how should these ids be added into Mondo).

**Comment by nicolevasilevsky** [comment]
there is this issue too:

FAIL Rule ../sparql/qc/general/qc-xref-syntax.sparql: 1 violation(s)
entity,property,value
_:b0,http://www.geneontology.org/formats/oboInOwl#hasDbXref,

I'm not sure which term is causing this failure though.

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -82,6 +82,7 @@ property_value: IAO:0000700 MONDO:0000001
...
...99/02/22-rdf-syntax-ns#>)\nPrefix(xml:=<http://www.w3.org/XML/1998/namespace>)\nPrefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)\nPrefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)\n\n\nOntology(\nDeclaration(Class(<http://purl.obolibrary.org/obo/MONDO_0005734>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/NCBITaxon_9793>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/NCBITaxon_9796>))\nDeclaration(ObjectProperty(<http://purl.obolibrary.org/obo/RO_0002162>))\nDeclaration(AnnotationProperty(<http://www.geneontology.org/formats/oboInOwl#source>))\n\n\n\n############################\n#   Classes\n############################\n\n# Class: <http://purl.obolibrary.org/obo/MONDO_0005734> (dourine)\n\nSubClassOf(Annotation(<http://www.geneontology.org/formats/oboInOwl#source> \"https://orcid.org/0000-0002-4142-7153\"^^xsd:string) Annotation(<http://www.geneontology.org/formats/oboInOwl#source> \"https://orcid.org/0000-0002-5002-8648\"^^xsd:string) <http://purl.obolibrary.org/obo/MONDO_0005734> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002162> ObjectUnionOf(<http://purl.obolibrary.org/obo/NCBITaxon_9793> <http://purl.obolibrary.org/obo/NCBITaxon_9796>)))\n\n\n)
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:85]: Something wrong happened here. Best guess: Problem with nesting in the OWL class expression of MONDO_0005734

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -99770,6 +99792,8 @@ xref: MESH:D004859 {source="MONDO:equivalentTo", source="EFO:0007263"}
 xref: UMLS:C0014661 {source="MONDO:equivalentTo"}
 is_a: MONDO:0005108 {source="EFO:0007263", source="MESH:D004859/inferred"} ! viral infectious disease
 is_a: MONDO:0005583 {source="https://orcid.org/0000-0001-5493-2602"} ! non-human animal disease
+relationship: in_taxon NCBITaxon:9788 {source="https://orcid.org/0000-0002-4142-7153", source="https://orcid.org/0000-0002-5002-8648", source="https://www.merckvetmanual.com/"} ! Equidae
+property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/4420 xsd:string
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:99796]: anyUri, not xsd:string

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -99957,6 +99981,8 @@ xref: EFO:0007277 {source="MONDO:equivalentTo"}
 xref: MESH:D005536 {source="MONDO:equivalentTo", source="EFO:0007277"}
 xref: UMLS:C0016514 {source="MONDO:equivalentTo"}
 is_a: MONDO:0005583 {source="EFO:0007277", source="MESH:D005536"} ! non-human animal disease
+property_value: RO:0002175 NCBITaxon:9823 {source="ISBN-10. 0323088392", source="ISBN-13. 978-0323088398", source="https://orcid.org/0000-0002-4142-7153", source="https://orcid.org/0000-0002-5002-8648"}
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:99984]: ISBN whitespace

Comment (matentzn) [reply src/ontology/mondo-edit.obo:99984]: and colob instead of .

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -419916,6 +419978,7 @@ name: parturient paresis
 def: "A disease of pregnant and lactating cows and ewes leading to generalized paresis and death. The disease, which is characterized by hypocalcemia, occurs at or shortly after parturition in cows and within weeks before or after parturition in ewes." [MESH:D010319]
 synonym: "animal milk fever" RELATED [MESH:D010319]
 synonym: "fever, animal milk" RELATED [MESH:D010319]
+synonym: "milk fever" EXACT [ISBN-10. 0323088392, ISBN-13. 978-0323088398]
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:419981]: Bandly formatted ISBN will have caused the error. `ISBN-10:0323088392` if ISBN-10 is a permitted prefix

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/ontology/mondo-edit.obo [hunk 6]
```diff
@@ -52938,6 +52942,7 @@ xref: SCTID:78714004 {source="DOID:5112"}
 xref: UMLS:C0038981 {source="DOID:5112", source="MONDO:notFoundInDiseaseSubset"}
 is_a: MONDO:0006873 {source="DOID:5112", source="MESH:D013540"} ! nutritional deficiency disease
 is_a: MONDO:0024985 {source="MESH:D013540"} ! sheep disease
+property_value: RO:0002175 NCBITaxon:9940 {source="ISBN-10. 0323088392", source="ISBN-13. 978-0323088398", source="https://orcid.org/0000-0002-4142-7153", source="https://orcid.org/0000-0002-5002-8648"}
```
[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:52945]: updated

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:99984]: updated

File: src/ontology/mondo-edit.obo (hunk 5 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:419981]: updated

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:99796]: updated

**Comment by sabrinatoro** [comment]
@nicolevasilevsky and @matentzn 
I need help with this (please and thank you!). I am unable to save changes in protege. I get the following error message:
java.lang.NullPointerException: null
	at org.obolibrary.obo2owl.OWLAPIObo2Owl.translateShorthandIdToExpandedId(OWLAPIObo2Owl.java:1739) ~[na:na]
	at org.obolibrary.obo2owl.OWLAPIObo2Owl.oboIdToIRI_load(OWLAPIObo2Owl.java:1679) ~[na:na]
	at org.obolibrary.obo2owl.OWLAPIObo2Owl.oboIdToIRI(OWLAPIObo2Owl.java:1643) ~[na:na]
	at org.obolibrary.oboformat.writer.OBOFormatWriter$OWLOntologyNameProvider.getName(OBOFormatWriter.java:992) ~[na:na]
	at org.semanticweb.owlapi.oboformat.OBOFormatRenderer$1.getName(OBOFormatRenderer.java:74) ~[na:na]
	at org.obolibrary.oboformat.writer.OBOFormatWriter.write(OBOFormatWriter.java:622) ~[na:na]
	at org.obolibrary.oboformat.writer.OBOFormatWriter.write(OBOFormatWriter.java:351) ~[na:na]
	at org.obolibrary.oboformat.writer.OBOFormatWriter.write(OBOFormatWriter.java:197) ~[na:na]
	at org.semanticweb.owlapi.oboformat.OBOFormatRenderer.render(OBOFormatRenderer.java:88) ~[na:na]
	at org.semanticweb.owlapi.oboformat.OBOFormatStorer.storeOntology(OBOFormatStorer.java:42) ~[na:na]
	at org.semanticweb.owlapi.util.AbstractOWLStorer.store(AbstractOWLStorer.java:99) ~[na:na]
	at org.semanticweb.owlapi.util.AbstractOWLStorer.storeOntology(AbstractOWLStorer.java:64) ~[na:na]
	at uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl.saveOntology(OWLOntologyManagerImpl.java:1486) ~[na:na]
	at uk.ac.manchester.cs.owl.owlapi.OWLImmutableOntologyImpl.saveOntology(OWLImmutableOntologyImpl.java:1459) ~[na:na]
	at uk.ac.manchester.cs.owl.owlapi.concurrent.ConcurrentOWLOntologyImpl.lambda$saveOntology$12(ConcurrentOWLOntologyImpl.java:369) ~[na:na]
	at uk.ac.manchester.cs.owl.owlapi.concurrent.ConcurrentOWLOntologyImpl.callReadLock(ConcurrentOWLOntologyImpl.java:196) ~[na:na]
	at uk.ac.manchester.cs.owl.owlapi.concurrent.ConcurrentOWLOntologyImpl.saveOntology(ConcurrentOWLOntologyImpl.java:369) ~[na:na]
	at org.protege.editor.owl.model.io.OntologySaver.saveOntologyInternal(OntologySaver.java:96) ~[na:na]
	at org.protege.editor.owl.model.io.OntologySaver.lambda$saveOntologies$0(OntologySaver.java:52) ~[na:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[na:1.8.0_121]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_121]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_121]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_121]

Note that I had to resolve some conflicts and merged with master, which might have been the cause of the issue?

(I obviously need more hand-holding when things like this happen, because I don't know what to do, and I don't understand the suggestions from slack)

**Comment by sabrinatoro** [comment]
work was done in another PR

**Pull request closed without merging**